dotcom-rendering/src/web/components/Nav/ExpandedMenu/Column.tsx

```diff
@@ -121,6 +124,15 @@ const pillarColumnLinks = css`
 `;
 
 const hideStyles = (columnInputId: string) => css`
+	${until.desktop} {
+		/* stylelint-disable-next-line selector-type-no-unknown */
+		${`#${columnInputId}`}:checked ~ & {
+			display: none;
+		}
+	}
+`;
+
+const notHideStyles = (columnInputId: string) => css`
```

Comment:
could we rename `hideStyles` and `notHideStlyes` to something like `hideWhenChecked` and `hideWhenNotChecked` or similar? I think this could be a bit more intuitive to read!

Reply:
Thanks for taking a look at this @OllysCoding. I have implemented all of your suggestions.